1.7/usage/tutorials/kafka/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 post_title: How to use Apache Kafka on DC/OS
 nav_title: Kafka
+menu_order: 07.5
```

Comment:
'tis. Not that it's best practice. I use it when it would be too much of a pain to renumber a whole menu. We've been incrementing menus by 10 to leave a little space between and avoid this inconvenience.